src/core/SignDoc.ts

```diff
@@ -42,7 +44,7 @@ export class SignDoc extends JSONSerializable<
       account_number,
       sequence,
       tx_body: { memo, messages, timeout_height },
-      auth_info: { fee },
+      auth_info: { fee, tip },
     } = this;
 
```

Comment:
It looks need to this logic
https://github.com/cosmos/cosmos-sdk/blob/7009a2e0cdb93b286199bbaa22e8880aeec83c8c/x/auth/tx/legacy_amino_json.go#L54-L78